posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -218,12 +218,6 @@ def results(self, columns: list[str], query_results: list):
                             "aggregations": self.extract_aggregations(result_dict),
                         }
                     )
-                else:
-                    logger.error(
-                        "error tracking issue not found",
-                        issue_id=result_dict["id"],
-                        exc_info=True,
-                    )
```

Comment:
It is now a valid case that we do not have an associated relational record for the query